dashboard/app/views/pd/application/teacher1920_application_mailer/confirmation.html.haml

```diff
@@ -25,6 +25,10 @@
   Or, visit
   = link_to 'https://code.org/educate', 'https://code.org/educate'
   for additional resources and opportunities to connect with other computer science educators.
+
+%p
+  = render partial: 'twitter_share'
```

Comment:
Quick clarification - this is the email we send to confirm that we received their application, but it does not mean they have been accepted. Do we want to use the same text in the tweet "I just signed up for Code.org's Professional Learning Program" if they haven't been accepted yet?

Reply:
Good catch.  Yes, this was discussed and approved.